frontend/service/agents/AgentsFunCelo.ts

```diff
@@ -0,0 +1,258 @@
+import { ethers } from 'ethers';
+import { formatEther } from 'ethers/lib/utils';
+
+import { STAKING_PROGRAMS } from '@/config/stakingPrograms';
+import { CELO_STAKING_PROGRAMS } from '@/config/stakingPrograms/celo';
```

Comment:
re above, just maybe need to make this abstract and make chainId param required everything instead of falling back to default